nfd-worker-daemonset.yaml.template

```diff
@@ -187,4 +195,25 @@ data:
     #            vendor: ["15b3"]
     #            device: ["1014", "1017"]
     #          loadedKMod : ["vendor_kmod1", "vendor_kmod2"]
+    #    - name: "feature.by.hostname"
+    #      value: customValue
+    #      matchOn:
+    #        - hostname: ["worker-0", "my-.*-node"]
 ### <NFD-WORKER-CONF-END-DO-NOT-REMOVE>
+---
+## Example for more custom configs in an additional configmap (3/3)
```

Comment:
Could this be a stand alone file, sameas the `nfd-worker-conf.example` file?

Reply:
Mm, why not. At least, for consistency, this example configmap should be added to the "combined" and "job" templates, too